cfme/cloud/instance/__init__.py

```diff
@@ -128,7 +128,7 @@ def is_displayed(self):
             relationship_provider_name = relationships.get_text_of('Cloud Provider')
         except (NameError, NoSuchElementException):
             logger.warning('No "Cloud Provider" Relationship, assume instance view not displayed')
-            return False
+            raise NotImplementedError("This view has no unique markers for is_displayed check")
```

Comment:
Please refer https://github.com/ManageIQ/integration_tests/pull/7989  for an exception `displayed_not_implemented`

Reply:
@digitronik I don't think that property will be useful here, as this is a special case where our ability to identify the screen is dynamic.

In this case we'll need to `raise NotImplementedError` directly.

Reply:
:+1: I  thought better to return `displayed_not_implemented`